app/Providers/InsightlyServiceProvider.php

```diff
@@ -4,12 +4,17 @@
 
 namespace App\Providers;
 
+use App\Domain\Contacts\Events\ContactCreated;
+use App\Domain\Integrations\Events\IntegrationCreated;
 use App\Insightly\InsightlyClient;
+use App\Insightly\Listeners\CreateContact;
+use App\Insightly\Listeners\CreateOpportunity;
 use App\Insightly\Pipelines;
 use GuzzleHttp\Client;
+use Illuminate\Support\Facades\Event;
 use Illuminate\Support\ServiceProvider;
 
-final class AppServiceProvider extends ServiceProvider
+final class InsightlyServiceProvider extends ServiceProvider
```

Comment:
To be consistent this can also be moved to the Insightly namespace

Reply:
Done